react/src/app.jsx

```diff
@@ -1,28 +1,27 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
 import { OverallRatings } from './overallRatings.jsx';
-import { Review } from './review.jsx';
+import { ReviewsList } from './reviewsList.jsx';
```

Comment:
You could get rid of a lot of the brackets if your components are `export default <ComponentName>`, since you only have one component per source file.  Non-blocking change suggestion

Reply:
thanks Mark. I made the refactors!